tools/e2etesting/steps/variables.yml

```diff
@@ -1,6 +1,6 @@
 variables:
   BasePath: $(System.DefaultWorkingDirectory)\tools\e2etesting
-  ClientCredentialsKeyVaultName: automatedtesting
+  ClientCredentialsKeyVaultName: developPipelineKV
```

Comment:
I'm not 100% confident of the use of this. Let me know and I can remove it.

Reply:
I agree that we should remove it.